openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/ProfilerDashboard.tsx

```diff
@@ -0,0 +1,353 @@
+        url: getDatabaseDetailsPath(fqn),
+      },
+      {
+        name: getPartialNameFromTableFQN(fqn, [FqnPart.Schema]),
+        url: getDatabaseSchemaDetailsPath(fqn),
+      },
+      {
+        name: getEntityName(table),
+        url: getTableTabPath(table.fullyQualifiedName || ''),
+      },
+      {
+        name: columnName,
+        url: '',
+        activeTitle: true,
+      },
+    ];
+  }, [table]);
+
+  const extraInfo: Array<ExtraInfo> = useMemo(() => {
+    return [
+      {
+        key: 'Owner',
+        value:
+          table.owner?.type === OwnerType.TEAM
+            ? getTeamAndUserDetailsPath(table.owner?.name || '')
+            : getEntityName(table.owner),
+        placeholderText: getEntityPlaceHolder(
+          getEntityName(table.owner),
+          table.owner?.deleted
+        ),
+        isLink: table.owner?.type === OwnerType.TEAM,
+        openInNewTab: false,
+        profileName:
+          table.owner?.type === OwnerType.USER ? table.owner?.name : undefined,
+      },
+      {
+        key: 'Tier',
+        value: tier?.tagFQN ? tier.tagFQN.split(FQN_SEPARATOR_CHAR)[1] : '',
+      },
+      { key: 'Type', value: `${table.tableType}`, showLabel: true },
+      {
+        value:
+          getUsagePercentile(
+            table.usageSummary?.weeklyStats?.percentileRank || 0,
+            true
+          ) || '--',
+      },
+      {
+        value: `${
+          table.usageSummary?.weeklyStats?.count.toLocaleString() || '--'
+        } queries`,
+      },
+    ];
+  }, [table]);
+
+  const onOwnerUpdate = (newOwner?: Table['owner']) => {
+    if (newOwner) {
+      const updatedTableDetails = {
+        ...table,
+        owner: {
+          ...table.owner,
+          ...newOwner,
+        },
+      };
+      onTableChange(updatedTableDetails);
+    }
+  };
+
+  const onTierUpdate = (newTier?: string) => {
+    if (newTier) {
+      const tierTag: Table['tags'] = newTier
+        ? [
+            ...getTagsWithoutTier(table.tags as Array<EntityTags>),
+            {
+              tagFQN: newTier,
+              labelType: LabelType.Manual,
+              state: State.Confirmed,
+            },
+          ]
+        : table.tags;
+      const updatedTableDetails = {
+        ...table,
+        tags: tierTag,
+      };
+
+      return onTableChange(updatedTableDetails);
+    } else {
+      return Promise.reject();
+    }
+  };
+
+  /**
+   * Formulates updated tags and updates table entity data for API call
+   * @param selectedTags
+   */
+  const onTagUpdate = (selectedTags?: Array<EntityTags>) => {
+    if (selectedTags) {
+      const updatedTags = [...(tier ? [tier] : []), ...selectedTags];
+      const updatedTable = { ...table, tags: updatedTags };
+      onTableChange(updatedTable);
+    }
+  };
+
+  const unfollowTable = async () => {
+    try {
+      const data = await removeFollower(table.id, getCurrentUserId());
+      const { oldValue } = data.changeDescription.fieldsDeleted[0];
```

Comment:
Can we get the current user and use the id from that and not use `getCurrentUserId` method?

Reply:
done